python/lsst/sims/maf/stackers/generalStackers.py

```diff
@@ -195,3 +193,44 @@ def run(self, simData):
             simData['gRGB'][match] = self.filter_rgb_map[f][1]
             simData['bRGB'][match] = self.filter_rgb_map[f][2]
         return simData
+
+
+class SeasonStacker(BaseStacker):
+    """
+    Add an integer label to show which season a given visit is in.
+    The season only depends on the RA of the object: we compute the MJD
+    when each object is on the meridian at midnight, and subtract 6
+    months to get the start date of each season.
+    The season index range is 0-10.
+    Must wrap 0th and 10th to get a total of 10 seasons.
+    """
+    def __init__(self, expMJDCol='expMJD',RACol='fieldRA'):
+        # Names of columns we want to add.
+        self.colsAdded = ['year', 'season']
+        # Names of columns we need from database.
+        self.colsReq = [expMJDCol, RACol]
+        # List of units for our new columns.
+        self.units = ['','']
+        # And save the column names.
+        self.expMJDCol = expMJDCol
+        self.RACol = RACol
+
+    def _run(self, simData):
+        # Define year number: (note that opsim defines "years" in flat 365 days).
+        year = np.floor((simData[self.expMJDCol] - simData[self.expMJDCol][0]) / 365)
```

Comment:
Never mind.  I thought you were dealing in absolute years and months, rather than delta_year, and delta_month.